features/LIBRARY.feature

```diff
@@ -0,0 +1,93 @@
+Feature: LIBRARY
+  Scenario: Editing books in the table
+    Given We are logged in as admin and we are in the LIBRARY section
+    When We click the edit button
+    And We change the title and / or author
+    And Confirm with the Ok button
+    Then The author and / or title for this book are changed
+
+  Scenario: Adding books to the table
+    Given We are logged in as admin and we are in the LIBRARY section
+    When We click the Add book button
+    And We complete the form
+    And Confirm with the Ok button
+    Then The book was added to the table
+
+  Scenario: Delete books from the table
+    Given We are logged in as admin and we are in the LIBRARY section
+    When We click the delete button
+    Then The book has been removed from the table
+
+  Scenario: Sorting alphabetically by title in ascending order
+    Given We are logged in and we are in the LIBRARY section
+    When Click on the drop-down list next to SORT BY and select Title ascending
```

Comment:
Jeśli masz ochotę użyć słowo and, to znaczy że warto je wpisać w nowym kroku. Jako konkretny krok który wykonujesz. Bo jest to coś co wykonujesz, a nie mieści się pod pierwszą częścią zdania tylko trzeba to osobno zapisać.